File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcCoordinateOperation/IfcMapConversion/IfcMapConversion.vb
Public Class IfcMapConversion Inherits IfcCoordinateOperation
    Public Eastings As IfcLengthMeasure
    Public Northings As IfcLengthMeasure
    Public OrthogonalHeight As IfcLengthMeasure
    Public XAxisAbscissa As IfcReal
    Public XAxisOrdinate As IfcReal
    Public Scale As IfcReal

    ' === EXTENDED BY ===
    ' IfcMapConversionScaled

    ' === WHERE CLAUSES ===
    ' TargetCRSOnlyProjected : 'IFC4X3_DEV_73740fe4.IFCPROJECTEDCRS' IN TYPEOF(SELF\IfcCoordinateOperation.TargetCRS)
End Class

File: src/main/resources/output/IFC4X3_python/Entities/IfcCoordinateOperation/IfcMapConversion/IfcMapConversion.py
class IfcMapConversion(IfcCoordinateOperation):
    def __init__(self):
        self.Eastings: IfcLengthMeasure = None
        self.Northings: IfcLengthMeasure = None
        self.OrthogonalHeight: IfcLengthMeasure = None
        self.XAxisAbscissa: IfcReal = None
        self.XAxisOrdinate: IfcReal = None
        self.Scale: IfcReal = None

    # === EXTENDED BY ===
    # IfcMapConversionScaled

    # === WHERE CLAUSES ===
    # TargetCRSOnlyProjected : 'IFC4X3_DEV_73740fe4.IFCPROJECTEDCRS' IN TYPEOF(SELF\IfcCoordinateOperation.TargetCRS)
